doc/ale.txt

```diff
@@ -216,6 +216,18 @@ g:ale_echo_cursor                                           *g:ale_echo_cursor*
   error at a column nearest to the cursor when the cursor is resting on a line
   which contains a warning or error. This option can be set to `0` to disable
   this behaviour.
+  The format of the message can be customizable in |g:ale_echo_msg_format|.
+
+
+g:ale_echo_msg_format                                    *g:ale_echo_msg_format*
+
+  Type: |String|
+  Default: `%s`
+
+  This variable defines the format of the echoed message. The `%s` is the
+  error message itself, and it can contain the following handlers:
+  - `%linter%` for linter's name
+  - `%severity%` for the type of severity
```

Comment:
Could you add a paragraph here with an example string for setting it up how you like it? I suspect others will want the same thing.